### PR TITLE
Qualcomm AI Engine Direct - add tutorial for op builder & quantizer

### PR DESCRIPTION
**Pull request by haowhsu-quic**
Summary
- tutorial for extending support of existent op builder & quantizer

### CODE REVIEW

File: backends/qualcomm/README.md [hunk 1]
```diff
@@ -73,3 +73,67 @@ examples/qualcomm
...
+- **Verify Unit Test Results**:<br/>
+    ```bash
+    cd $PATH_TO_EXECUTORCH
+    # example usage of performing unit test
+    python backends/qualcomm/tests/test_qnn_delegate.py -k TestQNNQuantizedOperator.test_qnn_backend_layer_norm -s $DEVICE_SERIAL -m SM8650 -b build-android/ -a $PATH_TO_TEST_ARTIFACTS
+    ```
+    The test graph is expected to have 1 delegated node with only placeholders / output nodes being left. Check the execution report for more information.
+
+- **Code Reviews**:<br/>
+    Please ping authors in Qualcomm AI Engine Direct related PRs for reviewing, possible candidates are listed below:
```
Comment (cccclai) [inline backends/qualcomm/README.md:131]: And add qualcomm tag so it's easier to track

**Review by cccclai — APPROVED** [review]
Thank you for adding the instruction to let users to add ops themselves!

File: backends/qualcomm/README.md (hunk 1 above)
Comment (haowhsu-quic) [reply backends/qualcomm/README.md:131]: Will update in other PR, thank you.

**Pull request closed without merging**